script/Ciel/Day 5/05_01_CIEL05_4B.txt

```diff
@@ -277,7 +277,8 @@ As if to show me the way―――
 [sha:ea258243fef18a7fb2ac628f3120d44d17cbca18]{
 -- Page 749, Offset 12888.
 -- 「“その遠吠えは　万里を手繰る―――！”」
-// There lore context to inform this, or just chuuni speak?
+// No lore context here, just chuuni speak
+// Flag: different bunk for 万里
```

Comment:
I think that "may ten thousand miles come to naught" works in english to as "make big small"

Reply:
"The howling... resounds across the beyond"